src/main/java/com/google/devtools/build/lib/analysis/PlatformOptions.java

```diff
@@ -80,6 +80,7 @@
 
   @Option(
       name = "platforms",
+      //allowMultiple = true,
```

Comment:
Is this commented out code intentional? If yes please include a TODO explaining its purpose.

Reply:
I initially added it, then realized it changes semantics and broke tests, and forgot to completely remove. Now it's really gone.